indico/modules/events/registration/models/items.py

```diff
@@ -437,6 +438,10 @@ def view_data(self):
     def _get_default_log_data(self):
         return {'Section ID': self.id}
 
+    def can_see(self, is_management):
```

Comment:
We typically pass the user explicitly to this kind of methods instead of accessing `session.user` in there. That's also nicer for testing etc.

Reply:
Updated, session.user was used to avoid adding 'user' when calling several templates.

Reply:
Why not just use `session.user` directly in the template when calling `can_see()`? No need to pass it around all the time.